PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/configuration/RemoteDebugSettingsEditor.java

```diff
@@ -42,6 +58,17 @@ protected void applyEditorTo(@NotNull RemoteDebugRunConfiguration remoteDebugRun
     @NotNull
     @Override
     protected JComponent createEditor() {
-        return new JPanel();
+
+        Project project = myRunConfiguration.getProject();
+        List<IClusterDetail> cachedClusterDetails = ClusterManagerEx.getInstance().getClusterDetailsWithoutAsync(true, project);
```

Comment:
>getClusterDetailsWithoutAsync [](start = 83, length = 29)

getClusterDetailsWithoutAsync is just getting all cached clusters. Is it on purpose?